tests/test_on_data/test_gin.py

```diff
@@ -0,0 +1,156 @@
+import tempfile
+import unittest
+from pathlib import Path
+import numpy.testing as npt
+import os
+
+import pytest
+from spikeextractors import NwbRecordingExtractor, NwbSortingExtractor
+from spikeextractors.testing import check_recordings_equal, check_sortings_equal
+from nwb_conversion_tools import (
+    NWBConverter,
+    IntanRecordingInterface,
+    NeuralynxRecordingInterface,
+    NeuroscopeRecordingInterface,
+    OpenEphysRecordingExtractorInterface,
+    PhySortingInterface,
+    SpikeGadgetsRecordingInterface,
+    SpikeGLXRecordingInterface,
+    BlackrockRecordingExtractorInterface,
+    BlackrockSortingExtractorInterface,
+)
+
+try:
+    from parameterized import parameterized, param
+
+    HAVE_PARAMETERIZED = True
+except ImportError:
+    HAVE_PARAMETERIZED = False
+
+# Path to dataset downloaded from https://gin.g-node.org/NeuralEnsemble/ephy_testing_data
+#   ecephys: https://gin.g-node.org/NeuralEnsemble/ephy_testing_data
+#   ophys: TODO
+#   icephys: TODO
+
+if os.getenv("CI"):
+    LOCAL_PATH = "."
+    print("running on Github CI")
+else:
+    LOCAL_PATH = Path(".")  # Must be set to "." for CI - temporarily override for local testing
+    print("running locally")
```

Comment:
```suggestion
if os.getenv("CI"):
    LOCAL_PATH = Path(".")  # Must be set to "." for CI
    print("Running GIN tests on Github CI!")
else:
    LOCAL_PATH = Path(".")  # Override this on personal device for local testing
    print("Running GIN tests locally!")
```
I do wish there was a better way to see these outputs other than sifting through the std callout's but better than nothing.

Thanks for finding this natural environment variable in the GitHub CI; that is a bit nicer for local override convenience!

Reply:
@luiztauffer I need to commit this change specifically ASAP because there are a bunch of PRs that branched from this and they're all failing because the CI path is a string, not `PathType`.